src/constants/index.js

```diff
@@ -77,3 +77,42 @@ export const PORTFOLIO_CONTENTS = [
 		video: "/assets/temp-picture.png",
 	},
 ];
+
+/**
+ * Skill
+ */
+import {
+	SiHtml5,
+	SiCss3,
+	SiJavascript,
+	SiReact,
+	SiNextdotjs,
+	SiNodedotjs,
+	SiMaterialui,
+	SiGit,
+	SiGithub,
+} from "react-icons/si";
+export const SKILL_TITLE = "My Skills";
+export const SKILL_CONTENTS = ["html", "css", "javascript", "react", "next", "node", "materialui", "git", "github"];
+export const SKILLS_ICON_NAMES = {
+	html: "HTML",
+	css: "CSS",
+	javascript: "JS",
+	react: "React",
+	next: "Next",
+	node: "Node",
+	materialui: "MUI",
+	git: "Git",
+	github: "GitHub",
+};
+export const SKILL_ICONS = {
+	html: <SiHtml5 />,
+	css: <SiCss3 />,
+	javascript: <SiJavascript />,
+	react: <SiReact />,
+	next: <SiNextdotjs />,
+	node: <SiNodedotjs />,
+	materialui: <SiMaterialui />,
+	git: <SiGit />,
+	github: <SiGithub />,
+};
```

Comment:
```suggestion
export const SKILL_DETAILS = {
  html: {
    name: "HTML",
    icon: <SiHtml5 />,
  },
};
```
Personally, I would prefer an object structure like this for those constants because that looks more readable to me. 
But it'll end up adding another object nesting when you use the constant, which could make it less simple to use at the same time.

I just wanted to share my thought so, you can ignore this suggestion if that doesn't bother you! 👍

Reply:
Thank you for your precise comments. I love your code, it is really easy to read.
So I'll change to it👍